.fig/autocomplete/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "boilerplate-fig-autocomplete",
+  "version": "1.0.0",
+  "description": "Boilerplate Fig autocomplete spec package",
```

Comment:
My thought is no, because this is actually a package.json for the [Fig dev tools](https://fig.io/docs/guides/private-autocomplete), not our own package.

open to changing it though if you think we should